### ISSUE TITLE
Update to 1.20

### ISSUE THREAD

**Issue by rdrpenguin04** [SquidDev-CC/CC-Tweaked#1475]
Alright; I have a few days where I can help with this. What needs to get done?

I've also heard that there's at least one change we were waiting on a major game version to update (Ender Modem Rebalancing); if we could compile a list of those, that could also be useful.

I'll make a PR for brunt-work porting in a little while (any obvious rename errors or similar); it would also be helpful to know if I should PR to this branch or to a different one.

Thank you all so much!

**Comment by SquidDev** [comment]
The initial update is already done - https://github.com/cc-tweaked/CC-Tweaked/tree/mc-1.20.x. It's not officially released yet, but one can grab builds from CI in the meantime. 

As far as #955 goes, that's more blocked on by indecision rather than a Minecraft update. I'm still not sure I've a plan which isn't going to be a pain in the neck to use.
